pkg/cri/sbserver/image_pull.go

```diff
@@ -36,6 +36,7 @@ import (
 	"github.com/containerd/imgcrypt"
 	"github.com/containerd/imgcrypt/images/encryption"
 	imagespec "github.com/opencontainers/image-spec/specs-go/v1"
+	"go.opentelemetry.io/otel/attribute"
```

Comment:
Can we hide this under `tracing` package, so we don't import it everywhere?

Something like `span.AddAttribute(key, value)` may be ?

Reply:
I see your point. However we are also specifying the type when adding attributes(using the provided attribute.<type>func)

How about defining a new type in tracing.go? Something like:
```
import (
	"go.opentelemetry.io/otel/attribute"
)

type AttributeKV = attribute.KeyValue

var attrBool = attribute.Bool
var attrString = attribute.String
...
```

this way we will only need to import the tracing package.

Reply:
> However we are also specifying the type when adding attributes(using the provided attribute.func)

Right, we already have a func that converts `interface{}` to `attribute.KeyValue`, so my thinking was that we could just reuse it here too.

https://github.com/containerd/containerd/blob/4e3d7e0aba0ecee165e3d47dc71315a1d91d7c97/tracing/log.go#L71